README.rst

```diff
@@ -121,6 +121,17 @@ NOTE: This configuration option currently only works for Linux.
     # Set notification expiry to 10 seconds
     export AUTO_NOTIFY_EXPIRE_TIME=10000
 
+    
+**Notification Urgency Level**
+
+By default non zero exit codes lead to critical notifications.
+On some systems (KDE) critical notifications will never expire.
+You can override notification urgency by setting the environment variable ``"AUTO_NOTIFY_URGENCY"`` to ``"low"``, ``"normal"``, or ``"critical``.
```

Comment:
Can your remove the quotes around "AUTO_NOTIFY_URGENCY", "low", "normal" and "critical". Marking them as code with `` is enough